File: spa_commission/lib/models/register_data.dart
class RegisterData {
  String? Password;
  String? FirstName;
  String? LastName;
  String? IdCardNumber;
  String? Birthday;
  String? Nationality;
  String? Address;
  String? Province;
  String? Occupation;
  String? PhoneNumber;
  String? Email;
  String? LineId;
  String? WhatsAppId;
  String? CompanyTypeOfUsage;
  String? CompanyName;
  String? CompanyTaxId;
  String? CompanyAddress;
  String? Bank;
  String? BankAccountNumber;
  String? ProfilePath;
  String? IdCardPath;

  RegisterData(
      {this.Password,
      this.FirstName,
      this.LastName,
      this.IdCardNumber,
      this.Birthday,
      this.Nationality,
      this.Address,
      this.Province,
      this.Occupation,
      this.PhoneNumber,
      this.Email,
      this.LineId,
      this.WhatsAppId,
      this.CompanyTypeOfUsage,
      this.CompanyName,
      this.CompanyTaxId,
      this.CompanyAddress,
      this.Bank,
      this.BankAccountNumber,
      this.ProfilePath,
      this.IdCardPath});

  Map<String, dynamic> toJson() => {
        'Password': Password,
        'FirstName': FirstName,
        'LastName': LastName,
        'IdCardNumber': IdCardNumber,
        'Birthday': Birthday,
        'Nationality': Nationality,
        'Address': Address,
        'Province': Province,
        'Occupation': Occupation,
        'PhoneNumber': PhoneNumber,
        'Email': Email,
        'LineId': LineId,
        'WhatsAppId': WhatsAppId,
        'CompanyTypeOfUsage': CompanyTypeOfUsage,
        'CompanyName': CompanyName,
        'CompanyTaxId': CompanyTaxId,
        'CompanyAddress': CompanyAddress,
        'Bank': Bank,
        'BankAccountNumber': BankAccountNumber,
        'ProfilePath': ProfilePath,
        'IdCardPath': IdCardPath,
      };
}

File: WebApplication13/Models/Mobile/RegisterData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApplication13.Models.Mobile
{
    public class RegisterData
    {
        public string Token { get; set; }
        public string Password { get; set; }
        public string TitleName { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string IdCardNumber { get; set; }
        public string Birthday { get; set; }
        public string Nationality { get; set; }
        public string Address { get; set; }
        public string Province { get; set; }
        public string Occupation { get; set; }
        public string PhoneNumber { get; set; }
        public string Email { get; set; }
        public string LineId { get; set; }
        public string WhatsAppId { get; set; }
        public string CompanyName { get; set; }
        public string CompanyTypeOfUsage { get; set; }
        public string CompanyTaxId { get; set; }
        public string CompanyAddress { get; set; }
        public string Bank { get; set; }
        public string BankAccountNumber { get; set; }
        public string ProfilePath { get; set; }
        public string IdCardPath { get; set; }
    }
}